repose-aggregator/src/docs/asciidoc/filters/herp.adoc

```diff
@@ -16,7 +16,17 @@ Events are recorded even if the API request is rejected by Repose before reachin
 
 == Prerequisites & Postconditions
 === Required Headers
-This filter has no required headers.
+[cols="15,12,73", options="header"]
```

Comment:
It looked pretty ugly with `autowidth`, mostly because of the header names.  The first two columns were squished and required wrapping.  I thought it looked better this way.  I can always change back if not better?